src/Razor/Microsoft.NET.Sdk.Razor/src/build/netstandard2.0/LinkerWorkaround.xml

```diff
@@ -19,6 +13,8 @@
     <type fullname="System.ComponentModel.TimeSpanConverter" />
   </assembly>
 
+  <assembly fullname="System.Runtime.InteropServices.JavaScript" preserve="all" />
+
   <assembly fullname="System.Text.Json">
```

Comment:
We *might* be able to drop this now (as it was working around an issue in STJ that might be fixed). Perfectly fine if you don't want to mess with extra stuff in this PR of course though!

Reply:
I filed https://github.com/dotnet/aspnetcore/issues/23262 to see if we can get these resolved by the BCL.